Jenkinsfile

```diff
@@ -308,6 +309,47 @@ def buildDiffCoverage() {
     }
 }
 
+def buildPerformance() {
+  return {
+    // Select docker-cph-X.  We want docker, metal (brix) and only one executor
+    // (exclusive), if the machine changes also change REALM_BENCH_MACHID below
+    node('docker && brix && exclusive') {
+      getSourceArchive()
+
+      def buildEnv = buildDockerEnv('ci/realm-core:snapshot')
+      // REALM_BENCH_DIR tells the gen_bench_hist.sh script where to place results
+      // REALM_BENCH_MACHID gives the results an id - results are organized by hardware to prevent mixing cached results with runs on different machines
+      // MPLCONFIGDIR gives the python matplotlib library a config directory, otherwise it will try to make one on the user home dir which fails in docker
+      buildEnv.inside {
+        withEnv(["REALM_BENCH_DIR=${env.WORKSPACE}/test/bench/core-benchmarks", "REALM_BENCH_MACHID=docker-brix","MPLCONFIGDIR=${env.WORKSPACE}/test/bench/config"]) {
+          withCredentials([[$class: 'FileBinding', credentialsId: 'c0cc8f9e-c3f1-4e22-b22f-6568392e26ae', variable: 's3cfg_config_file']]) {
+              sh 's3cmd -c $s3cfg_config_file get s3://static.realm.io/downloads/core/core-benchmarks.zip core-benchmarks.zip'
```

Comment:
Double quotes are needed to interpolate Groovy strings. Single quotes won't do that so this works because the variable is an environment variable available to the script.